src/officehours_ui/static/css/main.css

```diff
@@ -554,6 +553,35 @@ h3.alert-heading {
 	color: #cecbc9;
 }
 
+
+.table-responsive {
+    scrollbar-width: thin;
+    scrollbar-color: rgba(0, 0, 0, .5) white;
+}
+
+/* Firefox */
+@-moz-document url-prefix() {
+    .table-responsive {
+        scrollbar-width: thin;
+        scrollbar-color: rgba(0, 0, 0, .5) white;
+    }
+}
```

Comment:
Do you get this to work in Firefox? I didn't seem to, even if I mess around with the system settings on my Mac. Also, it looks like this is double applying styles, are both necessary?

I guess my first thought would be to not try to use these if we can't get them to work. Ideally we would, but I'd rather not commit something that is potentially not helping.

Reply:
No, I couldn't get it to work either, I can remove the classes for Firefox that I couldn't get to work.